huxley/www/js/actions/DelegateActions.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2011-2016 Berkeley Model United Nations. All rights reserved.
+ * Use of this source code is governed by a BSD License (see LICENSE).
+ */
+
+'use strict';
+
+var ActionConstants = require('constants/ActionConstants');
+var Dispatcher = require('dispatcher/Dispatcher');
+
+var DelegateActions = {
+  deleteDelegate(delegate) {
+    Dispatcher.dispatch({
+      actionType: ActionConstants.DELETE_DELEGATE,
+      delegate: delegate
+    });
+  },
+
+  addDelegate(delegate) {
+    Dispatcher.dispatch({
+      actionType: ActionConstants.ADD_DELEGATE,
+      delegate: delegate
+    });
+  },
+
+  updateDelegate(delegateID, name, email, schoolID) {
+    Dispatcher.dispatch({
+      actionType: ActionConstants.UPDATE_DELEGATE,
+      delegateID: delegateID,
+      name: name,
+      email: email,
+      schoolID: schoolID
```

Comment:
I don't think schoolID should be an updateable field. Opens us up to bugs where delegates are in the wrong school. we should also make this immutable on the server side, if we haven't already.